BUILDING_ON_LINUX.md

```diff
@@ -4,27 +4,28 @@ Note on Qt version compatibility: If you are installing Qt from a package manage
 
 ## Ubuntu 18.04
 
-_most likely works the same for other Debian-like distros_
+_Most likely works the same for other Debian-like distros_
 
-1. Install dependencies `sudo apt install qttools5-dev qtmultimedia5-dev libqt5svg5-dev libboost-dev libssl-dev libboost-system-dev libboost-filesystem-dev cmake g++`
+1. Install all of the dependencies using `sudo apt install qttools5-dev qtmultimedia5-dev libqt5svg5-dev libboost-dev libssl-dev libboost-system-dev libboost-filesystem-dev cmake g++`
 
-### Through Qt Creator
+### Compiling through Qt Creator
 
-1. Install C++ IDE Qt Creator `sudo apt install qtcreator`
+1. Install C++ IDE Qt Creator by using `sudo apt install qtcreator`
 1. Open `chatterino.pro` with Qt Creator and select build
 
 ### Manually
 
-1. go into project directory
-1. create build folder `mkdir build && cd build`
+1. Go into the project directory
+1. Create a build folder and open it (`mkdir build && cd build`)
+1. Use one of the options below to compile it
 
-#### Using QMake
+### Using CMake
 
-1. `qmake .. && make`
+`cmake .. && make`
 
-#### Using CMake
+### Using QMake
 
-1. `cmake .. && make`
+`qmake .. && make`
```

Comment:
Why was the order reversed? QMake is still the first building system that is used and CMake has been only added later. Imho this should stay as it and then maybe we'd just remove QMake altogether once we drop it.

Reply:
Described in PR description
 > -CMake is now always in front of QMake, should make users use CMake instead of QMake, since QMake is going to be deprecated soon. (Excluding the FreeBSD guide)

I think it's a fair reason